docs/guides/admin/docs/workflowoperationhandlers/select-streams-woh.md

```diff
@@ -0,0 +1,80 @@
+SelectStreamsWorkflowOperationHandler
+====================================
+
+
+Description
+-----------
+
+The SelectStreamsWorkflowOperationHandler can be used in case not all source tracks should be processed. For example,
+given a recording with a presenter and a presentation track, the final recording to be published should only include the
+video stream of the presenter track and the audio stream of the presentation track.
+
+The workflow operation will use workflow properties set by the Opencast video editor to determine which tracks should be
+selected for further processing and add them to the media package based on `target-flavor` and `target-tags`.
+
+
+Parameter Table
+---------------
+
+Configuration Key | Example   | Description
+:-----------------|:----------|:-----------
+source-flavor\*   | */source  | The flavor of the track(s) to use as a source input
+target-flavor\*   | */work    | The flavor of the target track(s)
+target-tags       | download  | The tags applied to all target tracks
+audio-muxing      | force     | Move single-audio media packages to a specific track (see below)
+force-target      | presenter     | Target track for the `force` setting for `audio-muxing`
+
+\* mandatory configuration key
+
+Audio Muxing
+-----------------
+
+The optional `audio-muxing` parameter has three possible values: `none` (same as omitting the option), `force` and
+`duplicate`.
+
+### `none` ###
+
+If `none` is specified or the option is omitted, tracks are taken from the specified `source-flavor` and are edited
+according to the selections in video editor’s “Tracks” panel. The resulting tracks are stored in the corresponding `
+target-flavor` and `target-tags` are applied.
+
+Editing in the “Tracks” panel usually means removing video or audio streams from tracks. However, if there are multiple
```

Comment:
I get what you're trying to say, but this is confusing.  What about something like:

Note: If your editing results in a single video and single audio (track/stream) they will be muxed together even if this option is set to `none`.